tensorflow_serving/servables/tensorflow/BUILD

```diff
@@ -199,6 +199,9 @@ cc_library(
     name = "serving_session",
     srcs = ["serving_session.cc"],
     hdrs = ["serving_session.h"],
+    visibility = [
+        "//visibility:public",
```

Comment:
I'm not sure we want to open up the visibility of serving_session and predict_impl. Do you need those to be open? If so, can you describe your use-case? Thanks.